exercise2.js

```diff
@@ -0,0 +1,19 @@
+//Decoder
+function solution (roman) {
+    const values = { I:1, V:5, X:10, L:50, C:100, D:500, M:1000 }
+    return roman.split("").reduce((tot,c,i,arr)=> tot + (values[c] < values[arr[i+1]] ? -values[c] : values[c]),0)
+}
+
+//Encoder
+
+function solution(number){
+  const values = { M:1000, CM:900, D:500, CD:400, C:100, XC:90, L:50, XL:40, X:10, IX:9, V:5, IV:4, I:1 };
+  const roman =""
+  for (let key in values){
+    while(number >=values[key]){
+        roman +=key
+        number -=values[key]
+    }
```

Comment:
Try to practice adding spaces around your loops to improve the readability of the code
```suggestion
  const values = { M:1000, CM:900, D:500, CD:400, C:100, XC:90, L:50, XL:40, X:10, IX:9, V:5, IV:4, I:1 };
  const roman =""
  
  for (let key in values){
    while(number >=values[key]){
        roman +=key
        number -=values[key]
    }
  }
 
```